Command/TestShortenerCommand.php

```diff
@@ -31,48 +46,47 @@ protected function configure()
 
         $this
             ->setDescription('Tests shortener')
-            ->setName('mremi:url-shortener:test');
+            ->setName('mremi:url-shortener:test')
+            ->addOption(
+                'provider',
+                null,
+                InputOption::VALUE_OPTIONAL,
```

Comment:
Should be `InputOption::VALUE_REQUIRED | InputOption::VALUE_IS_ARRAY`, so the code in `execute` becomes:
```php
$providers = $input->getOption('providers');

foreach ($chainProvider->getProviders() as $provider) {
    if ($providers && !in_array($provider, $providers)) {
        continue;
    }

    $link = $this->getLinkManager()->create();
    // ... existing code
}
```